react/tic-tac-toe/src/app/screens/Game/components/Board/index.js

```diff
@@ -1,19 +1,22 @@
 import React, { Component } from 'react';
-
 import Square from '../Square';
-
 import styles from './styles.module.scss';
 
 class Board extends Component {
-  renderSquare() {
-    return <Square />;
+
+  renderSquare(i) {
```

Comment:
Try to use a more descriptive variable name like `squarePosition` or something like that